cmd/helpMessages.go

```diff
@@ -134,162 +156,177 @@ const jobsCmdLongDescription = "Sub-commands related to managing jobs."
-  - file content:
-	dir1/dir2
-	blob1
-	blob2
 
-Remove a single file from ADLS Gen2 (include/exclude not supported):
-  - azcopy rm "https://[account].dfs.core.windows.net/[container]/[path/to/file]?[SAS]"
+   - azcopy rm "https://[account].blob.core.windows.net/[container]/[path/to/parent/dir]" --recursive=true --list-of-files=/usr/bar/list.txt
+   - file content:
+	   dir1/dir2
+	   blob1
+	   blob2
+
+Remove a single file from a Blob Storage account that has a hierarchical namespace (include/exclude not supported):
 
-Remove a single directory from ADLS Gen2 (include/exclude not supported):
-  - azcopy rm "https://[account].dfs.core.windows.net/[container]/[path/to/directory]?[SAS]"
+   - azcopy rm "https://[account].dfs.core.windows.net/[container]/[path/to/file]?[SAS]"
+
+Remove a single directory from a Blob Storage account that has a hierarchical namespace (include/exclude not supported):
+
+   - azcopy rm "https://[account].dfs.core.windows.net/[container]/[path/to/directory]?[SAS]"
 `
 
 // ===================================== SYNC COMMAND ===================================== //
 const syncCmdShortDescription = "Replicate source to the destination location"
 
 const syncCmdLongDescription = `
-Replicate a source to a destination location. The last modified times are used for comparison, the file is skipped if the last modified time in the destination is more recent. The supported pairs are:
+The last modified times are used for comparison. The file is skipped if the last modified time in the destination is more recent. The supported pairs are:
+  
   - local <-> Azure Blob (either SAS or OAuth authentication can be used)
   - Azure Blob <-> Azure Blob (Source must include a SAS or is publicly accessible; either SAS or OAuth authentication can be used for destination)
   - Azure File <-> Azure File (Source must include a SAS or is publicly accessible; SAS authentication should be used for destination)
 
-Please note that the sync command differs from the copy command in several ways:
-  0. The recursive flag is on by default.
-  1. The source and destination should not contain patterns(such as * or ?).
-  2. The include/exclude flags can be a list of patterns matching to the file names. Please refer to the example section for illustration.
-  3. If there are files/blobs at the destination that are not present at the source, the user will be prompted to delete them. This prompt can be silenced by using the corresponding flags to automatically answer the deletion question.
-  4. When syncing between virtual directories, add a trailing slash to the path (refer to examples) if there's a blob with the same name as one of the virtual directories 
+The sync command differs from the copy command in several ways:
+
+  1. The recursive flag is on by default.
+  2. The source and destination should not contain patterns (For example: * or ?).
```

Comment:
@zezha-msft this is incorrect information.
@normesta consider rewording this "sync can copy directories without the recursive flag, but will not sync sub-directories"

Reply:
The term "subdirectory" has no hyphen. Other than that, I've updated the text with your suggestion.

Reply:
Not sure what's happening here: the old # 1 (the one about source and destination should not contain patterns) is indeed obsolete and should have been removed.  Because now both copy and sync are the same in that regard. (I.e. it should not be the new # 2, it should be gone). 

But I think its important that the new # 1, should start with "The recursive flag is on by default", since this is indeed the opposite of what copy does.